python/yellowstone-fumarole-client/yellowstone_fumarole_client/__init__.py

```diff
@@ -85,10 +101,32 @@ class DragonsmouthAdapterSession:
     sink: asyncio.Queue
 
     # The queue for receiving SubscribeUpdate from the dragonsmouth stream.
-    source: asyncio.Queue
+    source: AsyncGenerator[SubscribeUpdate, None]
 
     # The task handle for the fumarole runtime.
-    fumarole_handle: asyncio.Task
+    _fumarole_handle: asyncio.Task
+
+    _sm: FumaroleSM = None
```

Comment:
I don't think it needs a default given that `DragonsmouthAdapterSession` is always instantiated with a state machine.